src/LaunchDarkly.Client/User.cs

```diff
@@ -151,7 +151,7 @@ public static User AndCustomAttribute(this User user, string attribute, int valu
             if (attribute == string.Empty)
                 throw new ArgumentException("Attribute Name can not be empty");
 
-            user.Custom.Add(attribute, new JValue(value));
+            user.Custom.Add(attribute, new JValue((double)value));
```

Comment:
Doesn't this mean we're definitively going to end up doing floating point comparisons?